Yank/pipeline.py

```diff
@@ -1112,6 +1111,43 @@ def _generate_molecule(self, molecule_id):
 
         return molecule
 
+    def _generate_residue_name(self, molecule_id):
```

Comment:
Can we leverage this to fix #465 while we are at it?

Reply:
That's actually a different problem that should be handled by `utils.TLeap`, but I can definitely make the change and add it to this PR!